docs/reference/spec/host-capabilities/06-kubernetes.md

```diff
@@ -122,3 +135,49 @@ Use the `list_resources_all` when dealing with cluster-wide resources.
 #### Output
 
 Result of `GET /apis/$api_version/namespaces/$namespace/$kind/$name`
+
+### Operation - `can_i`
+
+This operation determines if a specific user has the permission to perform an
+action on a Kubernetes resource. It does this by making a
+[`SubjectAccessReview`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.32/#subjectaccessreview-v1-authorization-k8s-io)
+request to the Kubernetes API server.
+
+#### Input
+
+The `can_i` operation requires a JSON object with the following parameters:
+
+```hcl
+{
+  # A JSON-serialized SubjectAccessReview object that defines the user, action, and resource to be checked.
+  "subject_access_review": {
+    "apiVersion": "authorization.k8s.io/v1",
+    "kind": "SubjectAccessReview",
+    "spec": {
+      "user": "system:serviceaccount:mynamespace:myuser",
+      "resourceAttributes": {
+        "namespace": "",
+        "verb": "list",
+        "resource": "secrets"
+      }
+    }
+  } ,
+  # Disable caching of results obtained from Kubernetes API Server
+  "disable_cache": bool
+}
+```
+
+#### Output
+
+The output is a JSON object that contains the status of the
+`SubjectAccessReviewi` request. This object indicates whether the requested
+action is allowed. For a complete reference of all the fields available in the
+output, see the official
+[`SubjectAccessReviewStatus`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.32/#subjectaccessreviewstatus-v1-authorization-k8s-io)
+documentation.
+
+```hcl
+{
+  "allowed": true
+}
+```
```

Comment:
I would like to add an example of a rejected request, to signify that the return is more complex than just a bool (can't suggest via UI, nested codeblocks). Something like:


Example of a rejected request:
```hcl
{
    "status": {
        "allowed": false,
        "denied": true,
        "reason": "User does not have permission to perform this action."
    }
}
```

Reply:
Let's make sure we explain the different meaning, since this is not something immediate to understand